include/integrations/smtp/smtp.php

```diff
@@ -200,6 +200,29 @@ public function saveOutgoingMailServerConfiguration(
 		$adb->pquery($sql, $params);
 	}
 
+	public function clearInc_SMTPSettings() {
+		global $adb, $current_user;
+		$result = $adb->pquery('select 1 from vtiger_mail_accounts where user_id=?', array($current_user->id));
+		if ($adb->num_rows($result) > 0) {
```

Comment:
you don't need to make this check, if you launch an update and no record is found, MySQL will simply do nothing. So with one query you have enough. There is no need to change this, it isn't wrong to check nor important in this case as the performance impact is insignificant here. This comment is just for your knowledge.